src/responder/kcm/kcmsrv_ccache.h

```diff
@@ -96,6 +103,10 @@ struct kcm_cred *kcm_cred_new(TALLOC_CTX *mem_ctx,
                               uuid_t uuid,
                               struct sss_iobuf *cred_blob);
 
+/* Create copy of cred structure. */
+struct kcm_cred *kcm_cred_dup(TALLOC_CTX *mem_ctx,
```

Comment:
This doesn't seem to be used outside of `kcmsrv_ccache.c`
I propose to make it static within the module.

Reply:
Done.